src/java.base/share/classes/java/lang/invoke/VarHandle.java

```diff
@@ -1636,6 +1636,10 @@ public boolean hasInvokeExactBehavior() {
         COMPARE_AND_EXCHANGE(Object.class),
         GET_AND_UPDATE(Object.class);
 
+        static final int COUNT = 5;
```

Comment:
```suggestion
        static final int COUNT = GET_AND_UPDATE.ordinal() + 1;
```

This would avoid the hardcoded count which is a bit fragile.